app/register/page.test.tsx

```diff
@@ -83,23 +92,78 @@ describe('Register', () => {
         password: 'rawr123',
         confirmPassword: 'rawr123',
       });
-      expect(mockLoginAccount).toHaveBeenCalledWith({
+      expect(mockLogin).toHaveBeenCalledWith({
         email: 'rt@example.com',
         password: 'rawr123',
         confirmPassword: 'rawr123',
       });
     });
   });
 
-  test('redirects to /weeklyPicks when the button is clicked', async () => {
+  test('redirects to /league/all when the button is clicked', async () => {
     mockUseAuthContext.isSignedIn = true;
 
     render(<Register />);
 
     await waitFor(() => {
-      expect(mockPush).toHaveBeenCalledWith('/weeklyPicks');
+      expect(mockPush).toHaveBeenCalledWith('/league/all');
     });
 
     mockUseAuthContext.isSignedIn = false;
   });
+
+  test('should show success notification upon successful submission', async () => {
+    fireEvent.change(emailInput, { target: { value: 'test@test.com' }});
+    fireEvent.change(passwordInput, { target: { value: 'pw1234' }});
+    fireEvent.change(confirmPasswordInput, { target: { value: 'pw1234' }});
+    fireEvent.click(continueButton);
+
+    await waitFor(() => {
+      expect(registerAccount).toHaveBeenCalledWith({
+        email: 'test@test.com',
+        password: 'pw1234',
+        confirmPassword: 'pw1234',
+      });
+      expect(mockLogin).toHaveBeenCalledWith({
+        email: 'test@test.com',
+        password: 'pw1234',
+        confirmPassword: 'pw1234',
+      });
+      expect(mockPush).toHaveBeenCalledWith('/league/all');
+      expect(toast.custom).toHaveBeenCalledWith(
+        <Alert
+          variant={AlertVariants.Success}
+          message="You have successfully registered your account."
+        />,
+      );
+    });
+  });
+
+  test('should show error notification upon submission failing', async () => {
+    mockLogin.mockImplementationOnce(() => Promise.reject(new Error('Mock error')));
+
+    fireEvent.change(emailInput, { target: { value: 'test@test.com' }});
+    fireEvent.change(passwordInput, { target: { value: 'pw1234' }});
+    fireEvent.change(confirmPasswordInput, { target: { value: 'pw1234' }});
+    fireEvent.click(continueButton);
+
+    await waitFor(() => {
+      expect(registerAccount).toHaveBeenCalledWith({
+        email: 'test@test.com',
+        password: 'pw1234',
+        confirmPassword: 'pw1234',
+      });
+      expect(mockLogin).toHaveBeenCalledWith({
+        email: 'test@test.com',
+        password: 'pw1234',
+        confirmPassword: 'pw1234',
+      });
+      expect(toast.custom).toHaveBeenCalledWith(
+        <Alert
+          variant={AlertVariants.Error}
+          message="Something went wrong!"
```

Comment:
Please sort alphabetically

Reply:
I feel the need to not alphabetize this as I always define the type of variant first before providing a custom message. The message is always based on the variant type.